Align CI and pipeline setup with behave-html-formatter

bittner opened the pull request:
Updates the CI and pipeline setup, supporting more current Python versions and sporting more linters.

See also: https://github.com/behave-contrib/behave-html-formatter/pull/34
Closes: #4

I've also take the freedom to reformat the code with isort and Black, to make sure the linting jobs all pass. Keeping the minify job identical made it necessary to also align the file names of the CSS theme and JS script files.

bittner committed:
Align CI and pipeline setup with behave-html-formatter

bittner committed:
Reformat module imports with isort

Executed as: `tox -e isort behave_html_pretty_formatter tests`

bittner committed:
Shorten lines (make both Flake8 and Black happy)

bittner committed:
Please Pylint, update supported Pythons

bittner commented:
All test and linting jobs pass, except for Pylint, which complains for the right, I think.

Please decide for yourselves, whether to split up the module into separate files (I would suggest so) or ignore the warnings via a directive in `pyproject.toml`.

fpokryvk commented on tox.ini:
```diff
@@ -80,10 +91,7 @@ html-pretty = behave_html_pretty_formatter:PrettyHTMLFormatter
-max-line-length = 100
-
-[pylint.main]
-disable = missing-module-docstring,too-many-lines,too-many-instance-attributes,too-many-arguments
```
We have changed which warnings we want to disable. Maybe we will split it into multiple files later on.

modehnal replied:
Yeah we can spit it later, no need to rush it for now.

fpokryvk reviewed (commented):
Thank you, I will change the pylint configuration to make the tests pass for now.

fpokryvk committed:
Change pylint configuration

We managed not to have some warning already disabled,
 ut some new appeared, we will deal with them in the future.

modehnal reviewed (dismissed):
Looks good to me, thank you.

modehnal committed:
Fix installation line

fpokryvk reviewed (approved):
Looks good, great work. Thank you all.

bittner commented on README.md:
````diff
@@ -6,7 +6,7 @@
 
 ```shell
-sudo python3 -m pip install behave-html-pretty-formatter
+python3 -m pip install behave-html-pretty-formatter
````
:+1:
